src/oscillator.js

```diff
@@ -3,6 +3,41 @@ import Add from 'Tone/signal/Add';
 import Mult from 'Tone/signal/Multiply';
 import Scale from 'Tone/signal/Scale';
 
+// ========================== //
+// SIGNAL MATH FOR MODULATION //
+// ========================== //
+
+// return sigChain(this, scale, thisChain, nextChain, Scale);
+class sigChain {
```

Comment:
I'm not sure if `sigChain` should be a class, since we're not calling it with the `new` keyword, it seems like it can just remain a function.

Reply:
calling a method like `add` that calls `sigChain` results in

```
Uncaught TypeError: Cannot call a class as a function
```